e2e/test/helpers/templates/PoolingOverAmqp.cs

```diff
@@ -80,10 +78,14 @@ internal static class PoolingOverAmqp
                 // Close the service-side components and dispose the device client instances.
                 if (cleanupOperation != null)
                 {
-                    await cleanupOperation().ConfigureAwait(false);
+                    try
+                    {
+                        await cleanupOperation().ConfigureAwait(false);
+                    }
+                    catch { }
```

Comment:
Anything specific you are aware of that is caught here?

Reply:
Nope, just making sure cleanup doesn't fail the test.